doc/source/tutorial/neighborfinding.rst

```diff
@@ -0,0 +1,217 @@
+For instance, in a binary system we might be interested in the :py:class:`freud.pmft.PMFTXYZ` of one particle type with respect to the other.
+In such cases, we can use the ``query_points`` argument to perform this calculation:
+
+.. code-block:: python
+
+    pmft = freud.pmft.PMFTXYZ(rmax, rmax, rmax, nbins, nbins, nbins)
+    positions_A = ...
+    positions_B = ...
+
+    # For simplicity, assume all particles are isotropic (spherical) and
+    # therefore orientations are irrelevant.
+    orientations = np.array([[1, 0, 0, 0]*num_points)
+
+    pmft.compute((box, positions_A), orientations=orientations,
+                 query_points=positions_B, query_orientations=orientations)
+
+Let's take a moment to understand exactly what this calculation means.
+Internally, **freud** will create a :py:class:`freud.locality.NeighborQuery` object using ``(box, positions_A)``, and then call its ``queryBall`` method, passing in ``positions_B`` and ``rmax`` as the query arguments (default for PMFTs that can, of course, be overridden using the ``query_args``).
+The resulting sets of neighbors will then be used in the calculation of the PMFT.
+As a result, we will calculate the PMFT of the distribution of ``positions_A`` around centers located at ``positions_B``.
+
+Understanding this sequence is particularly important when using the ``nearest`` mode of neighbor finding, because *this mode is not symmetric*.
+To understand what this means, consider the following simple example:
+
+
+.. code-block:: python
+
+    positions_A = [[0, 0, 0]]
+    positions_B = [[-1, 0, 0], [1, 0, 0]]
+    nq_A = freud.locality.AABBQuery(box=box, points=positions_A)
+    count_A = 0
+    for _ in nq_A.query(positions_B, num_neighbors=1):
+        count_A += 1
+
+    nq_B = freud.locality.AABBQuery(box=box, points=positions_B)
+    count_B = 0
+    for _ in nq_B.query(positions_A, num_neighbors=1):
+        count_B += 1
+
+    print(count_A)
+    >>> 2
+
+    print(count_B)
+    >>> 1
+
+The reason these calculations give different results is simple.
+We only asked for one neighbor for each point, but in the first case we queried for two points, and as a result, it found us one neighbor for each of the two points.
+In the second case, we constructed our object with two points, but then only requested the neighbors for the single point in ``positions_A``; as a result, we only found one neighbor.
+This logic is precisely what governs the ``query_points`` argument; the ``points`` are used to build the query object, and the ``query_points`` are what is passed to the ``query*`` methods.
+
+We are now in a position to explain the ``exclude_ii`` query argument.
+When performing a calculation on a single-component system, we typically do not wish to include a particle as its own neighbor.
+The ``exclude_ii`` argument indicates to a ``query*`` method that any pair of particles with identical indices in the ``points`` and ``query_points`` arrays should be ignored.
```

Comment:
I think the glob syntax is potentially confusing, perhaps use plain English instead of code here.